bindings/python/src/OpenSpaceToolkitCorePy/Type/Real.cpp

```diff
@@ -2,15 +2,15 @@
 
 #include <OpenSpaceToolkit/Core/Type/Real.hpp>
 
-inline void OpenSpaceToolkitCorePy_Type_Real(pybind11::module& aModule)
+using ostk::core::type::Integer;
+using ostk::core::type::Real;
+using ostk::core::type::String;
+
+inline void OpenSpaceToolkitCorePy_Type_Real(pybind11::class_<Real>& realClass)
```

Comment:
Not fully following what's going on here, but will this work for modules that have sub-modules?

In some files like Phyics [Environment](https://github.com/open-space-collective/open-space-toolkit-physics/blob/main/bindings/python/src/OpenSpaceToolkitPhysicsPy/Environment.cpp#L198)

We need a reference to the module in order to instantiate sub-modules. Is it possible to still do this?